docker-compose-dev.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  gondolin-postgis:
+    image: mdillon/postgis:10-alpine
+    ports:
+      - "5555:5432"
+    environment:
+      POSTGRES_USER: gondolin
+      POSTGRES_PASSWORD: gondolin
+    volumes:
+      - ./postgresql_data:/var/lib/postgresql/data:Z
+  gondolin-pgadmin:
+    image: dpage/pgadmin4
+    ports:
+      - "5050:80"
+    environment:
+      PGADMIN_DEFAULT_PASSWORD: SuperSecret
+      PGADMIN_DEFAULT_EMAIL: user@domain.com
```

Comment:
You might want to change this to something different. Or even leave it empty if it is not required.

Reply:
For some reason it's required, but I changed it to match our default